calicoctl/resourcemgr/networkset.go

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2019 Tigera, Inc. All rights reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package resourcemgr
+
+import (
+	"context"
+
+	api "github.com/projectcalico/libcalico-go/lib/apis/v3"
+	client "github.com/projectcalico/libcalico-go/lib/clientv3"
+	"github.com/projectcalico/libcalico-go/lib/options"
+)
+
+func init() {
+	registerResource(
+		api.NewNetworkSet(),
+		api.NewNetworkSetList(),
+		true,
+		[]string{"networkset", "networksets", "ns"},
```

Comment:
Hrm, the abbreviation "ns" seems nice but is also what Kubernetes uses for "Namespace" so might be confusing... especially if we decide to add a Namespace object some day.

Maybe `netset`? WDYT?

Reply:
Ahh that's a good point. `netset` is nice. And for GlobalNetworkSets, `gnetset`?